README.md

```diff
@@ -31,7 +31,7 @@ A number of atlases are in development, but those available currently are:
 - [Gubra's LSFM mouse brain atlas](https://doi.org/10.1007/s12021-020-09490-8) at 20 micron resolution
 - [3D version of the Allen mouse spinal cord atlas](https://doi.org/10.1101/2021.05.06.443008) at 20 x 10 x 10 micron resolution
 - [AZBA: A 3D Adult Zebrafish Brain Atlas](https://doi.org/10.1101/2021.05.04.442625) at 4 micron resolution
-- [Waxholm Space atlas of the Sprague Dawley rat brain](https://doi.org/10.1016/j.neuroimage.2014.04.001) at 39 micron resolution
+- [Waxholm Space atlas of the Sprague Dawley rat brain](https://doi.org/10.1038/s41592-023-02034-3) at 39 micron resolution
```

Comment:
@PolarBean can I check that this new reference is the most appropriate one for the version of the atlas that we have in BrainGlobe? See https://github.com/brainglobe/brainglobe-atlasapi/issues/215 for context.

Reply:
thats is fair enough, I agree the updated citation would be more appropriate once you have the 4.01. that being said the newest publication is more similar to 4.0 than is Papp 2014.